src/canmatrix/formats/xlsx.py

```diff
@@ -418,14 +418,12 @@ def getIfPossible(row, value):
 
 
 def load(filename, **options):
-    from sys import modules
-
     # use xlrd excel reader if available, because its more robust
     if 'xlsxLegacy' in options and options['xlsxLegacy'] == True:
         logger.error("xlsx: using legacy xlsx-reader - please get xlrd working for better results!")
     else:
-        import canmatrix.xls
-        return canmatrix.xls.load(filename, **options)
+        import canmatrix.formats.xls as xls_loader  # we need alias, otherwise we hide the globally imported canmatrix
```

Comment:
or try `global canmatrix` if it helps?

Reply:
is ok so, just one (of many) refactoring failures from my side...